scripts/lib/create-production-venv

```diff
@@ -8,15 +8,31 @@ ZULIP_PATH = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__f
 if ZULIP_PATH not in sys.path:
     sys.path.append(ZULIP_PATH)
 
-from scripts.lib.zulip_tools import overwrite_symlink, run, subprocess_text_output
-from scripts.lib.setup_venv import setup_virtualenv, VENV_DEPENDENCIES
+from scripts.lib.zulip_tools import overwrite_symlink, run, parse_lsb_release
+from scripts.lib.setup_venv import (
+    setup_virtualenv, VENV_DEPENDENCIES, REDHAT_VENV_DEPENDENCIES,
+    FEDORA_VENV_DEPENDENCIES
+)
 
 parser = argparse.ArgumentParser(description="Create a production virtualenv with caching")
 parser.add_argument("deploy_path")
 args = parser.parse_args()
 
 # install dependencies for setting up the virtualenv
-run(["apt-get", "-y", "install"] + VENV_DEPENDENCIES)
+distro_info = parse_lsb_release()
+vendor = distro_info['DISTRIB_ID']
+family = 'redhat' if vendor in ['CentOS', 'Fedora', 'RedHat'] else 'debian'
```

Comment:
Let's just edit `parse_lsb_release` return to include a `DISTRIB_FAMILY` -- better to not copy this parsing logic in N places.